### DISCUSSION TITLE
Making Things Easier 2: Review Menu structure

### DISCUSSION THREAD

**Discussion by beccon4** [freeplane/freeplane#1147] [Ideas]
Freeplane is a really nice and useful program - but sometimes, the UI gets a bit lengthy and overcrowded.

- to insert e.g. a summary node - I use the context menu which is nice - but have to dive into the third level (Insert -> Add Node -> join nodes (*) )  
- there I can delete nodes too. Why would I want to search a "delete" option within an "insert" branch?
- "insert graphic" the same - but this sub menu only contains one useful item "insert graphic" and "delete graphic" grayed out.

The same for styles: I can easily apply styles with the context menu which goes very well - but to merely change the colour one has to go to the top as this is not part of the context menu...

Any ideas how one could tidy up this without breaking to much the known UI?

(*) translated back from German version

**Comment by Illegit** [comment]
I use the menu-o-matic addon of EdoFro to create a personalized menu. Have a look at the addons if it is matching your needs.
